test/sequential/test-http-server-unconsume-consume.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+const common = require('../common');
+const http = require('http');
+const assert = require('assert');
+
+if (process.argv[2] === 'child')
+  child();
+else
+  parent();
+
+function parent() {
```

Comment:
Why does it have to be this way? It is ok if the test crashes or exits with non-zero code. No need to wrap it up this way.